src/js/components/Pagination/__tests__/Pagination-test.js

```diff
@@ -345,4 +345,25 @@ describe('Pagination', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test(`should change the page on prop change`, () => {
+    const { container, rerender, getByText } = render(
+      <Grommet>
+        <Pagination numberItems={NUM_ITEMS} page={1} />
+      </Grommet>,
+    );
+
+    expect(getByText('1').className).not.toBe(getByText('3').className);
```

Comment:
Just a small suggestion to align with the convention of other tests in this file. We typically don't reference `className` In this case, maybe you could do something like the test above:
```
const desiredPage = '1';
const activePage = container.querySelector(`[aria-current="page"]`)
      .innerHTML;
expect(activePage).toEqual(desiredPage);

Reply:
Thanks. I made it a bit more concise, hope you don't mind. Let me know please if you think I should make it exactly as posted above.